pkg/flags/unique_urls_test.go

```diff
@@ -91,3 +93,29 @@ func TestNewUniqueURLsWithExceptions(t *testing.T) {
 		}
 	}
 }
+
+func TestUniqueURLsFromFlag(t *testing.T) {
+	const name = "test"
+	urls := []string{
+		"https://1.2.3.4:1",
+		"https://1.2.3.4:2",
+		"https://1.2.3.4:3",
+		"https://1.2.3.4:1",
+	}
+	fs := flag.NewFlagSet(name, flag.ExitOnError)
+	u := NewUniqueURLsWithExceptions(strings.Join(urls, ","))
+	fs.Var(u, name, "usage")
+	uss := UniqueURLsFromFlag(fs, name)
+
+	if len(uss) != len(u.Values) {
+		t.Fatal("not unique")
+	}
+
+	um := make(map[string]struct{})
+	for _, x := range uss {
+		um[x.String()] = struct{}{}
+	}
+	if !reflect.DeepEqual(um, u.Values) {
+		t.Fatal("not equal")
+	}
```

Comment:
```suggestion
	require.Equal(t, u.Values, um)
```

Reply:
It's just a minor comment,` require.Equal` is recommended. If you look at the implementation, it actually calls `reflect.DeepEqual`, but it's simpler from user perspective.

Reply:
I understand. 
But if a file has two styles of tests, it's not very comfortable.

Reply:
then pls change line 90 as well.

Reply:
Done.